.goreleaser.yml

```diff
@@ -0,0 +1,10 @@
+project_name: es-index-manager
+builds:
```

Comment:
This should configure goreleaser to create a GitHub release and changelog, but not build any artifacts. 

https://goreleaser.com/cookbooks/release-a-library/